.openpublishing.redirection.json

```diff
@@ -1022,6 +1022,14 @@
         {
             "source_path": "docs/csharp/programming-guide/strings/how-to-convert-rtf-to-plain-text.md",
             "redirect_url": "https://msdn.microsoft.com/en-us/library/cc488002(v=vs.140).aspx"
+        },
+        {
+            "source_path": "docs/csharp/parallel.md",
+            "redirect_url": "/dotnet/csharp/programming-guide/threading/index.md"
```

Comment:
I also don't think this is a good redirect. This is one of the examples of our triplication of documentation -- we have framework threading, C# threading, and VB threading, when there's no language support for threading. But since this was supposed to be a parallel programming topic, the best place to link anyway is [Parallel Programming in the .NET Framework](~/docs/standandard/parallel-programming/index.md)